src/components/Modal/__tests__/Modal.spec.js

```diff
@@ -7,23 +7,22 @@ const renderComponent = (options) => render(Modal, { ...options });
 describe('Modal', () => {
 
   it('is hidden by default', () => {
-    const { queryByRole } = renderComponent();
+    const slots = { header: 'modal header' };
+    const { queryByRole } = renderComponent({ slots });
 
     const modal = queryByRole('dialog');
     expect(modal).not.toBeInTheDocument();
-
   });
 
   it('is visible when its visible prop is true', () => {
     const props = {
       visible: true
     };
-
-    const { queryByRole } = renderComponent({ props });
+    const slots = { header: 'modal header' };
```

Comment:
I think you need to update these tests now

Reply:
will do as soon as I finish the Buttons renaming!